libpod/container_api.go

```diff
@@ -109,6 +107,34 @@ func (c *Container) Start() (err error) {
 		}
 	}()
 
+	// Reinitialize the container if we need to
+	if c.state.State == ContainerStateStopped {
+		// If necessary, delete attach and ctl files
+		if err := c.removeConmonFiles(); err != nil {
+			return err
+		}
+
+		// Delete the container in the runtime
+		if err := c.runtime.ociRuntime.deleteContainer(c); err != nil {
+			return errors.Wrapf(err, "error removing container %s from runtime", c.ID())
```

Comment:
Could this fail?  With the container does not exist.

Will this work, if I do

podman run ...
podman stop 
reboot
podmain start

Reply:
On reboot, we reset state to ContainerStateConfigured to reflect that we no longer exist in the runtime

Reply:
Ok, could you make sure that this scenario works correctly.  Perhaps have this as a suggested test or at least just rm -rf /run/runc/ and then start and make sure nothing goes wacky.